web/src/components/seo.js

```diff
@@ -50,15 +48,6 @@ const SEO = ({ title, description, image, article }) => {
                 <meta property="og:description" content={seo.description} />
             )}
             {seo.image && <meta property="og:image" content={seo.image} />}
-            <meta name="twitter:card" content="summary_large_image" />
-            {twitter && <meta name="twitter:creator" content={twitter} />}
-            {seo.title && <meta name="twitter:title" content={seo.title} />}
-            {seo.description && (
-                <meta name="twitter:description" content={seo.description} />
-            )}
-            {seo.image && (
-                <meta name="twitter:image" content={`${seo.image}?w=800`} />
-            )}
```

Comment:
I don't think you want to remove all of these, just this one:
`{twitter && <meta name="twitter:creator" content={twitter} />`